public/app/features/alerting/unified/components/rule-editor/PreviewRuleResult.tsx

```diff
@@ -37,7 +48,7 @@ export function PreviewRuleResult(props: Props): React.ReactElement | null {
       </div>
     );
   }
-
+  console.log(data);
```

Comment:
I think this one needs to be removed before we can merge :)

Reply:
ops